File: 8kyu/(8 kyu) Plural/(8 kyu) Plural.ts
// 1 Simple solution
export function plural(n: number): boolean {
    if (n === 1 || n === -1) {
        return false;
    } else {
        return true;
    }
}

// 2 Optimized solution
export function plural(n: number): boolean {
    return n !== 1 && n !== -1;
}

// 3 Coding golf
export const plural = (n: number) => n != 1;

File: 8kyu/(8 kyu) Plural/(8 kyu) Plural.cs
// 1 Simple solution
public class Kata
{
  public static bool Plural(double n)
  {
    if (n == 1 || n == -1)
    {
      return false;
    } else
    {
      return true;
    }
  }
}

// 2 Optimized solution
public class Kata
{
  public static bool Plural(double n)
  {
    return n != 1 && n != -1;
  }
}

// 3 Clever solution
public class Kata
{
  public static bool Plural(double n) => n != 1;
}
